projectmanagement/src/main/java/life/qbic/projectmanagement/application/ExperimentInformationService.java

```diff
@@ -0,0 +1,138 @@
+package life.qbic.projectmanagement.application;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import life.qbic.logging.api.Logger;
+import life.qbic.logging.service.LoggerFactory;
+import life.qbic.projectmanagement.domain.project.experiment.Experiment;
+import life.qbic.projectmanagement.domain.project.experiment.ExperimentId;
+import life.qbic.projectmanagement.domain.project.experiment.repository.ExperimentRepository;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Analyte;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Species;
+import life.qbic.projectmanagement.domain.project.experiment.vocabulary.Specimen;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service that provides an API to query basic project information
+ *
+ * @since 1.0.0
+ */
+@Service
+public class ExperimentInformationService {
+
+  private static final Logger log = LoggerFactory.logger(ExperimentInformationService.class);
+  private final ExperimentRepository experimentRepository;
+
+  public ExperimentInformationService(@Autowired ExperimentRepository experimentRepository) {
+    this.experimentRepository = experimentRepository;
+  }
+
+  @PostAuthorize("hasPermission(returnObject,'VIEW_EXPERIMENT')")
```

Comment:
why do we need that? Thats not a real permission that we have. Rather we have edit or create experiment as special permissions :)